pkg/instrumentation/podmutator.go

```diff
@@ -391,10 +392,26 @@ func (pm *instPodMutator) selectInstrumentationInstanceFromNamespace(ctx context
 	switch s := len(otelInsts.Items); {
 	case s == 0:
 		pm.Logger.Info("no OpenTelemetry Instrumentation instances available. Using default Instrumentation instance")
-		return getDefaultInstrumentation()
+		cr := GetAmazonCloudWatchAgentResource(pm.Client, "cloudwatch-agent")
```

Comment:
I can make `cloudwatch-agent` a const somewhere

Reply:
That makes sense